scripts/stage-release.sh

```diff
@@ -183,12 +191,14 @@ gh repo set-default "$(git remote get-url "${REPOSITORY}")"
 # create GitHub release via CLI https://cli.github.com/manual/gh_release_create
 # it is created as a draft, the deploy_release workflow will publish it.
 echo "Creating draft release notes."
-gh release create "${RELEASE_TAG}" --title "${RELEASE_TAG}" --notes-file "CHANGELOG.md" --draft
+# csplit will create a file for every version as we use ## to denote versions. We also use # CHANGELOG as a header so the current release is actually in the 01 file (zero based)
```

Comment:
Wondering if we can move the csplit command down here so it's close to the usage.

Also in the kroxylicious repo we have gitignored `.releaseNotes`.

Reply:
(edit: since we're trying to sync with kroxy, lets refactor later)